src/openforms/registrations/contrib/json_dump/config.py

```diff
@@ -85,3 +95,4 @@ class JSONDumpOptions(TypedDict):
     variables: list[str]
     fixed_metadata_variables: list[str]
     additional_metadata_variables: list[str]
+    transform_to_list: list
```

Comment:
```suggestion
    transform_to_list: list[str]
```
we know the items are strings